Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ghcr.io/steamdeckhomebrew/holo-base:latest as tools
```

Comment:
This entire dockerfile can probably be replaced by the image used in decky-plugin-database.

Reply:
you mean ghcr.io/steamdeckhomebrew/builder:latest, right? yeah, maybe I went too far with holoiso image. I'll try it with builder image and update PR if fits

Reply:
as for SHD's docker image — it can be used but needs a different entrypoint, there is no need for rsync when you're testing locally, not to mention it grabs a lot of unnecessary files. The existing build process is a bit messy but it'll do for now, I guess. I'll update PR a bit later, still need some testing

Reply:
updated, standing by for input from you guys